README.md

```diff
@@ -6,27 +6,78 @@
 
 # Quantum serverless
 
+Quantum Serverless is a user-friendly tool that enables you to easily run complex quantum computing tasks. 
+With this software, you can execute Qiskit programs as long running jobs and distribute them across multiple CPUs, GPUs, and QPUs. 
+This means you can take on more complex quantum-classical programs and run them with ease. 
+You don't have to worry about configuration or scaling up computational resources, as Quantum Serverless takes care of everything for you. 
+
 ![diagram](./docs/images/qs_diagram.png)
 
 ### Table of Contents
 
 1. [Installation](INSTALL.md)
 2. [Quickstart](#quickstart-guide)
 3. [Beginners Guide](docs/beginners_guide.md)
-4. Modules:
+4. [Getting started](docs/getting_started/)
+5. Modules:
    1. [Client](./client)
    2. [Infrastructure](./infrastructure)
-5. [Tutorials](docs/tutorials/)
-6. [Guides](docs/guides/)
-7. [How to Give Feedback](#how-to-give-feedback)
-8. [Contribution Guidelines](#contribution-guidelines)
-9. [References and Acknowledgements](#references-and-acknowledgements)
-10. [License](#license)
+6. [Tutorials](docs/tutorials/)
+7. [Guides](docs/guides/)
+8. [How to Give Feedback](#how-to-give-feedback)
+9. [Contribution Guidelines](#contribution-guidelines)
+10. [References and Acknowledgements](#references-and-acknowledgements)
+11. [License](#license)
 
 ----------------------------------------------------------------------------------------------------
 
 ### Quickstart
 
+
+
+<details>
+  <summary>1. Hello World quickstart</summary>
+
+1 - Create program file `hello_qiskit.py`
```

Comment:
Would it be worth it to have a `samples/` directory with these code snippets in it? so folks don't have to c&p...

Reply:
getting started is now referring to docker compose jupyter instance which has everything needed (all files and examples)